gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml

```
@@ -43,6 +43,54 @@ security:
     - CookieAuth: []
 
 paths:
+    /pages/_homepage:
```

```suggestion
    /portal-pages/_homepage:
```

I may be wrong but I think it has been discussed with florent and the team. On mapi and papi side it has been decided to use `/portal-pages/` 🤔 Let me validate it one last time and I come back to you 😅

---

as discussed, created a new use case and updated the endpoint